src/Components/Layouts/Pages/ProfilePage/ProfileUploads/UploadItem.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Grid from '@material-ui/core/Grid';
+import './ProfileUploads.css'
+
+const UploadItem = ({key, uploadURL, status}) => (
+    <Grid item xs={4} spacing={1}>
```

Comment:
Upload item as component shouldn't have width. It should be in parent markup.

Reply:
Depends on your monitor size🤪 I know it shouldn't be this way